core/src/de/adesso/brainysnake/Gamelogic/Player/PlayerController.java

```diff
@@ -70,10 +70,13 @@ public void updatePlayerState(GlobalGameState gameState) {
             } catch (InterruptedException e) {
                 LOGGER.error("PlayerController",
                         "Player: " + playerHandler.getPlayerName() + " got Timeout after " + Config.MAX_AGENT_PROCESSING_TIME_MS + " ms", e);
+                        e.printStackTrace();
             } catch (ExecutionException e) {
                 LOGGER.error("PlayerController: ", "ExecutionException - " + e.getMessage());
+                e.printStackTrace();
             } catch (TimeoutException e) {
-                LOGGER.error("PlayerController: ", "Waiting for Player " + playerHandler.getPlayerName() + " aborted. Timeout");
+                LOGGER.error("PlayerController: ", "Waiting for Player " + playerHandler.getPlayerName() + " aborted. Timeout" );
```

Comment:
Platzhalter {} verwenden
Bitte in die slf4j doku schauen

Reply:
done